internal/v1/handler.go

```diff
@@ -268,8 +268,8 @@ func (h *Handlers) GetComposeStatus(ctx echo.Context, composeId uuid.UUID) error
 		return err
 	}
 	if composeRequest.Customizations != nil && composeRequest.Customizations.Users != nil {
-		for _, u := range *composeRequest.Customizations.Users {
-			u.RedactPassword()
+		for i := range *composeRequest.Customizations.Users {
+			(*composeRequest.Customizations.Users)[i].RedactPassword()
 		}
 	}
```

Comment:
Out of curiosity - why this change? I do not have my IDE opened but `Users` is a pointer I assume. Therefore, even when `u` is a copy it should not matter. Does it work when you leave the shorter version?

But yeah, sometimes you need to do this in Go since `u` is a copy. To come up with a shorter version this is cleaner:

```
users := *composeRequest.Customizations.Users
for i := range users {
	users[i].RedactPassword()
}
```

Reply:
yes, `TestComposeStatus` was failing without this change, shorter change is nice